master/internal/restore.go

```diff
@@ -340,3 +341,62 @@ func shimExperimentSnapshotV2(snapshot []byte) ([]byte, error) {
 
 	return json.Marshal(experimentSnapshotV2)
 }
+
+// Version 3 => 4 had no shim, it was breaking.
+
+// Version 4 => 5 shims
```

Comment:
stray comment?  Or what is this saying?

Reply:
eh, i'll remove these if they're confusing at all